src/common/snippets/include/snippets/lowered/loop_info.hpp

```diff
@@ -145,6 +145,16 @@ class LoopInfo {
     const char* get_type_name() const {
         return get_type_info().name;
     }
+    /**
+     * @brief Return true if expression port is a loop port
+     * @param expr_port - expression port to check
+     */
+    bool is_loop_port(const ExpressionPort& expr_port);
+    /**
+     * @brief Return loop port of an expression port
+     * @param expr_port - expression port.
+     */
+    LoopPort get_loop_port(const ExpressionPort& expr_port);
```

Comment:
Minor: by the way, can we return const ref here?

Reply:
done, thanks Alexandra!